.drone.star

```diff
@@ -2261,6 +2242,7 @@ def setupNotificationsAppForServer():
             "rm -rf %s/apps/notifications" % dir["server"],
             "git clone -b master https://github.com/owncloud/notifications.git %s/apps/notifications" % dir["server"],
             "cd %s || exit" % dir["server"],
+            "php occ upgrade",
```

Comment:
This is to fix the error while setting up the oauth app https://drone.owncloud.com/owncloud/web/37247/43/16